### PR TITLE
fix: move bundlephobia call in the backend

### LINKED ISSUE #91: fix(http-server): CORS issue issue https://bundlephobia.com
# Current behavior:

Bundlephobia is currently made on the front end side
https://github.com/ES-Community/nsecure/blob/v0.9.0/public/js/utils.js#L217

# Issue:
We have a CORS issue then.

# 🚀 Solution:

Create an endpoint `packages/bundle?name=<name>&version=<version>` and make the request there.

### PR DESCRIPTION
**Pull request by tony-go**
Hi 👋 

This issue resolves #91 

Added:
- `bundle/:pkgName/:version` endpoint
- `bundle/:pkgName/` endpoints
- functional tests
- add storage for HTTP server port

> Note: we could switch from `bundle/:pkgName/:version` to `bundle/:pkgName?version=:v` if you prefer.

### CODE REVIEW

**Comment by tony-go** [comment]
Note: the UI was a bit broken when I try to test in the UI

File: public/js/utils.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { getJSON } from "@nodesecure/vis-network";
 
 // Import static
 import avatarURL from "../img/avatar-default.png";
+import { portStore } from "../../src/http-server/context";
```
Comment (fraxken) [inline public/js/utils.js:7]: You'r not on the server here but on front (it will not work).

File: public/js/utils.js [hunk 2]
```diff
@@ -212,9 +213,10 @@ export function copyToClipboard(str) {
 
 export async function getBundlephobiaSize(name, version) {
   try {
+    const port = portStore.getStore();
     const {
       gzip, size, dependencySizes
-    } = await getJSON(`https://bundlephobia.com/api/size?package=${name}@${version}`);
+    } = await getJSON(`http://localhost:${port}/bundle/${name}/${version}`);
```
Comment (fraxken) [inline public/js/utils.js:219]: It is not necessary to know the port here. Just request `/bundle/${name}/${version}` (fetch by default will use the localhost with the current port).

Comment (tony-go) [reply public/js/utils.js:219]: Wasn't sure about that thanks a lot ^^ I'll clean it a bit.

File: public/js/utils.js (hunk 1 above)
Comment (tony-go) [reply public/js/utils.js:7]: Yeah 🤦!

**Pull request merged**